### DISCUSSION TITLE
Dinit on NixNG

### DISCUSSION THREAD

**Discussion by RafTeog** [davmac314/dinit#407] [General]
Hey, I am having a blast with dinit on artix. Thank you very much for this amazingly simple software.
Good news, I just asked in nixos forum, and it seems that there will be dinit integration in a nixos community based distro.
https://discourse.nixos.org/t/restructuring-nixos-to-work-without-systemd-e-g-with-sysvinit/21298/51

Nice!
Cheers

**Comment by davmac314** [comment]
Interesting, thanks for the link. :)
